src/TerminalVelocityChen2022.jl

```diff
@@ -0,0 +1,111 @@
+module TerminalVelocityChen2022
+
+import SpecialFunctions as SF
+import Thermodynamics as TD
+
+import ..Common as CO
+import ..CommonTypes as CT
+
+import ..Parameters as CMP
+const APS = CMP.AbstractCloudMicrophysicsParameters
+
+export rain_terminal_velocity_individual
+export rain_terminal_velocity_bulk
+
+"""
+    rain_terminal_velocity_individual(ρ, scheme, D_r)
+
+ - `ρ`` - air density
+ - `scheme` - type for parameterization
+ - `D_r` - diameter of the raindrops
+
+ Returns the fall velocity of a raindrop using multiple gamma-function terms
+ to increase accuracy for `scheme == Chen2022Type`
+"""
+function rain_terminal_velocity_individual(
```

Comment:
I would suggest defining that one in the docs example. It's not going to be used in the actual source code. We just need the parameters to compute the bulk group velocity

Reply:
Sounds good, will do!